Bash/InstallHuntress-macOS-bash.sh

```diff
@@ -61,27 +60,50 @@ install_system_extension=true
 ## Do not modify anything below this line
 ##############################################################################
 
-scriptVersion="March 25, 2025"
 
-dd=$(date "+%Y%m%d-%H%M%S")
+scriptVersion="April 14, 2025"
+
+version="1.1 - $scriptVersion"
+dd=$(date "+%Y-%m-%d  %H:%M:%S")
 log_file="/tmp/HuntressInstaller.log"
+log_file_location="/Users/Shared/"
 install_script="/tmp/HuntressMacInstall.sh"
 invalid_key="Invalid account secret key"
 pattern="[a-f0-9]{32}"
-version="1.1 - March 25, 2025"
 
-## Using logger function to provide helpful logs within RMM tools in addition to log file
+# Using logger function to provide helpful logs within RMM tools in addition to log file
 logger() {
     echo "$dd -- $*";
     echo "$dd -- $*" >> $log_file;
 }
 
+# Copies the log from a temp location to /users/shared/  and exits with the given code 
+# Using this folder as /tmp/ is wiped on reboot, Huntress folders are protected by TP, and because any user should have access to this folder
+copyLogAndExit() {
+    local exitCode="$1"
+    if [ -d $log_file_location ]; then
+        logger "Copying log file to /Users/Shared/"
+        cp "$log_file" $log_file_location"HuntressInstaller.log"    
```

Comment:
```suggestion
        cp "$log_file" "${log_file_location}/HuntressInstaller.log"    
```

Fixed quotation marks and variable expansion next to slash